src/index.js

```diff
@@ -44,7 +46,12 @@ const bindEvents = function CozyBarBindEvents () {
   document.body.addEventListener('click', this._clickOutsideListener)
 
   this._drawerObserver = this.observe('drawerVisible', drawerVisible => {
-    document.querySelector('[role=banner]').dataset.drawerVisible = drawerVisible
+    const bannerNode = document.querySelector(BANNER_SELECTOR)
+    if (!bannerNode) {
+      console.warn(`Cozy-bar is looking for a "${BANNER_SELECTOR}" tag that contains your banner and can't find it :'(…`)
```

Comment:
This observer is bound only if bar iew is created (see https://github.com/cozy/cozy-bar/pull/25/files#diff-1fdf421c05c1140f6d71444ea2b27638R106), so this condition is probably useless